### PR TITLE
feat(storybook): add rtl/ltr toggle storybook addon

### PR DESCRIPTION
**Pull request by marcosmoura**
This PR adds a button to toggle from LTR to RTL in Storybook examples, when running the dev server:

image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Review by dmytrokirpa — APPROVED** [review]

File: packages/react-components/react-storybook-addon/package.json [hunk 1]
```diff
@@ -30,20 +30,21 @@
     "@fluentui/scripts-tasks": "*"
```
Comment (fabricteam) [inline packages/react-components/react-storybook-addon/package.json:30]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32814&commitId=a414d1c7248b3fef1eed245787e0f2b48b096efa&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|8| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay High Contrast.chromium.png|2251| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-storybook-addon/src/components/DirSwitch.styles.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { makeStyles } from '@griffel/react';
```
Comment (Hotell) [inline packages/react-components/react-storybook-addon/src/components/DirSwitch.styles.ts:1]: not sure this was the best architectural decision , for storybook UIs we should stick with storybook styling approach

File: packages/react-components/react-storybook-addon/src/preset/manager.ts [hunk 3]
```diff
@@ -12,6 +13,13 @@ addons.register(ADDON_ID, () => {
     match: ({ viewMode }) => !!(viewMode && viewMode.match(/^(story|docs)$/)),
     render: ThemePicker,
   });
+  addons.add(DIR_ID, {
+    title: 'Dir Switch',
```
Comment (Hotell) [inline packages/react-components/react-storybook-addon/src/preset/manager.ts:17]: it would be good to provide human readable names, Direction rather `Dir`

**Comment by Hotell** [comment]
now we have 2 implementations to do 1 thing, wondering how feasible is to unite the logic under 1 thing ( web-components ) folks also have it

https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/DocsComponents/DirSwitch.stories.tsx

File: packages/react-components/react-storybook-addon/src/preset/manager.ts (hunk 3 above)
Comment (marcosmoura) [reply packages/react-components/react-storybook-addon/src/preset/manager.ts:17]: Will be fixed in follow up PR

**Pull request merged**